src/pipeline.h

```diff
@@ -226,6 +226,24 @@ class Pipeline {
   /// Generates a default depth attachment in D32_SFLOAT_S8_UINT format.
   std::unique_ptr<Buffer> GenerateDefaultDepthAttachmentBuffer() const;
 
+  /// Information on values set for OpenCL-C plain-old-data args.
+  struct ArgSetInfo {
+    std::string name = "";
```

Comment:
I think "" is the default for std::string anyway?

Reply:
Just wanted to be specific I guess.

Reply:
Yes, members with default constructors are default-constructed.